packages/wrangler/src/dev.tsx

```diff
@@ -391,13 +397,36 @@ function useCustomBuild(
       stderr: "inherit",
       stdout: "inherit",
     });
+    if (watch_dir) {
+      watch(watch_dir, { persistent: true, ignoreInitial: true }).on(
+        "all",
+        (_event, _path) => {
+          console.log("Restarting build...");
```

Comment:
Perhaps log the path of the changed file that caused the rebuild?

Something like?

```ts
console.log(`The file ${path} changed`);
```

Reply:
done and done